src/main/java/io/anserini/search/SearchTweets.java

```diff
@@ -145,13 +146,14 @@ public static void main(String[] args) throws Exception {
       RerankerContext context = new RerankerContext(searcher, query, topic.getId(), topic.getQuery(),
          queryTokens, FIELD_BODY, filter);
       ScoredDocuments docs = cascade.run(ScoredDocuments.fromTopDocs(rs, searcher), context);
+      long qtime = (System.currentTimeMillis() - startQueryTime);
```

Comment:
`qtime` isn't great, `elapsed_time` or `query_time` would be better, even if it is relatively obvious what's being calculated. Also, check `startQueryTime` matches the start time you mean 😄  (my last comment might have been a bit misleading because I missed that there were two timers going on)

And because I can't comment above -- `long curQueryTime = System.nanoTime()` should be switched to `currentTimeMillis` to match.